pkg/apis/trebuchet/install/install.go

```diff
@@ -0,0 +1,12 @@
+package install
+
+import (
+	"github.com/atlassian/voyager/pkg/apis/creator/v1"
```

Comment:
This should be trebuchet api group
```suggestion
	trebuchet_v1 "github.com/atlassian/voyager/pkg/apis/trebuchet/v1"
```